larq/layers.py

```diff
@@ -52,7 +70,10 @@ def call(self, inputs):
             inputs = self.input_quantizer(inputs)
         if self.kernel_quantizer:
             full_precision_kernel = self.kernel
-            self.kernel = self.kernel_quantizer(self.kernel)
+            quantized_kernel = self.kernel_quantizer(self.kernel)
+            if hasattr(self, "mean_changed_values"):
+                self.add_metric(self.mean_changed_values(quantized_kernel))
```

Comment:
Could you perhaps add a brief comment on how this works?

Reply:
It adds the metric to a layer the same way an accuracy metric is added to a model in when passing it to `keras.models.Model.compile` (If I recall correctly, `keras.models.Model` also inherits from `keras.layers.Layer`):

https://github.com/tensorflow/tensorflow/blob/0bbeb15d2247f2fae98cbec4e5a93c285ed5ab0c/tensorflow/python/keras/engine/base_layer.py#L1026-L1042